### PR TITLE
Changelog 28

### PR DESCRIPTION
**Pull request by deltaepsilon**
image

### CODE REVIEW

File: docs-content/general/changelog/changelog-28.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+If those buttons are active, you can click through to see the related resource and dig deeper into the error's cause.
+
+And of course, we have similar buttons on our Session Replay page, as well as links back to the relevant session from the Traces page.
+
+![Related Resources buttons](/images/changelog/28/related-resources.png)
+
+## Search query evolution
+
+Search is large problem set, and we're continuing to chip away at it.
```
Comment (Vadman97) [inline docs-content/general/changelog/changelog-28.md:22]:
```suggestion
Search is a large problem set, and we're continuing to chip away at it.
```

**Review by Vadman97 — APPROVED** [review]
🚢

File: highlight.io/public/images/changelog/28/log-with-trace-id.png
Comment (SpennyNDaJets) [inline highlight.io/public/images/changelog/28/log-with-trace-id.png:1]: Wanted to call out that the log with related trace is actually this "Related trace" button here
Screenshot 2024-03-06 at 11 15 09 AM

File: docs-content/general/changelog/changelog-28.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+We're tightening up that integrated approach by detecting those connections proactively. Now, when you're investigating an error, you'll see buttons for related sessions, logs, and traces.
+
+If those buttons are active, you can click through to see the related resource and dig deeper into the error's cause.
+
+And of course, we have similar buttons on our Session Replay page, as well as links back to the relevant session from the Traces page.
+
+![Related Resources buttons](/images/changelog/28/related-resources.png)
+
+## Search query evolution
```
Comment (SpennyNDaJets) [inline docs-content/general/changelog/changelog-28.md:20]: Not sure if we want to call out these short cuts on the query search - contains and matches (i.e. regex)
Screenshot 2024-03-06 at 11 17 15 AM

**Review by SpennyNDaJets — APPROVED** [review]
LGTM! Just maybe the one photo change?

File: docs-content/general/changelog/changelog-28.md (hunk 2 above)
Comment (deltaepsilon) [reply docs-content/general/changelog/changelog-28.md:20]: Thanks for the images!

**Pull request merged**